doc/src/stdlib/linalg.md

```diff
@@ -73,6 +73,10 @@ Base.LinAlg.diagm
 Base.LinAlg.scale!
 Base.LinAlg.rank
 Base.LinAlg.norm
+Base.LinAlg.norm(::AbstractVector, ::Real)
```

Comment:
I hope Michael comes back, we miss you!

In the meantime if he's still traveling, maybe @mortenpi can give input on Documenter issues